sql/analyzer/pushdown.go

```diff
@@ -33,7 +33,36 @@ func pushdownFilters(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, se
 		return n, transform.SameTree, nil
 	}
 
-	return pushdownFiltersAtNode(ctx, a, n, scope, sel)
+	node, same, err := pushdownFiltersAtNode(ctx, a, n, scope, sel)
+	var hasBindVars bool
+	transform.Inspect(n, func(node sql.Node) bool {
```

Comment:
you use this elsewhere, maybe a good candidate for a helper method `filtersHasBindvar`, or something similar